templates/lxc-download.in

```diff
@@ -48,17 +46,17 @@ DOWNLOAD_USE_CACHE="false"
 DOWNLOAD_VALIDATE="true"
 DOWNLOAD_VARIANT="default"
 
+DOWNLOAD_KEYID="0xE7FB0CAEC8173D669066514CBAEFF88C22F6E216"
+if [[ -z "$DOWNLOAD_KEYSERVER" ]]; then
```

Comment:
Ah - I had not noticed it was POSIX mode, and yes the var check is not good there. Pushed changes.